src/main/java/net/starschema/clouddb/jdbc/BQConnection.java

```diff
@@ -164,6 +166,10 @@ public BQConnection(String url, Properties loginProp) throws SQLException {
         String withServiceAccountParam = caseInsensitiveProps.getProperty("withserviceaccount");
         Boolean serviceAccount = (withServiceAccountParam != null) && Boolean.parseBoolean(withServiceAccountParam);
 
+        // There's really no reason anyone would ever not want to use this... but leave it in for a bit as a bailout switch
+        String useQueryApiParam = caseInsensitiveProps.getProperty("usequeryapi");
+        useQueryApi = useQueryApiParam == null || !useQueryApiParam.equalsIgnoreCase("false");
```

Comment:
Does `!useQueryApiParam.equalsIgnoreCase("false")` not cover the null case as well?